grails-app/views/stockMovement/show.gsp

```diff
@@ -90,7 +90,11 @@
             <g:isUserInAllRoles location="${stockMovement?.origin?.id}" roles="${[RoleType.ROLE_REQUISITION_APPROVER]}">
                 <g:set var="userHasRequestApproverRole" value="${true}"/>
             </g:isUserInAllRoles>
-            <g:set var="isApprovalRequired" value="${stockMovement?.requisition?.approvalRequired && stockMovement?.origin?.isApprovalRequired()}" />
+            <g:set var="isApprovalRequired"
+                   value="${stockMovement?.requisition?.approvalRequired &&
+                           stockMovement?.origin?.isApprovalRequired() &&
+                           stockMovement.status < RequisitionStatus.PICKING}"
```

Comment:
I feel like this is dangerous - won't it compare the statuses chronologically?

Reply:
Are you sure? it's `compare` method which is not the same as `compareTo` method available in `Comparator` interface. Those are not the same.

Reply:
@alannadolny yup please make sure about that one

Reply:
This topic was previously answered by Justin: https://github.com/openboxes/openboxes/pull/4133#discussion_r1255253493